app.py

```diff
@@ -57,6 +57,7 @@ def update_cfg_by_env(cfg):
     env_model_settings['base_path'] = os.getenv('S3_BASE_PATH')
     env_model_settings['access_key_id'] = os.getenv('S3_ACCESS_KEY')
     env_model_settings['secret_access_key'] = os.getenv('S3_SECRET_KEY')
+    env_model_settings['force_sync'] = os.getenv('RWS_FORCE_SYNC') == 'True' or False
```

Comment:
Usually any non-empty value of an environment variable is considered as true. It is quite usual to set `FOO=1`. If we'll interpret only `True` as truth, it'll be confusing (especially, if we'll do it case sensitively).

Reply:
I don't want "False" to mean "True". But I agree that validation can be case insensitive. I will also add an additional note to README.

Reply:
Updated.

Reply:
I don't not want to make `RWS_FORCE_SYNC=1` means `False`.

I experimented around etcd's option `ETCD_ENABLE_V2`. My results:

| environment | result |
| ---- | ---- |
| ETCD_ENABLE_V2=1 | enabled |
| ETCD_ENABLE_V2=true | enabled |
| ETCD_ENABLE_V2=false | disabled |
| ETCD_ENABLE_V2=FALSE | disabled |
| ETCD_ENABLE_V2=0 | disabled |

How to check:

1. Run `ETCD_ENABLE_V2=<...> etcd` in a temporary directory.
2. Run `curl http://127.0.0.1:2379/v2/keys/message`
    - `404 page not found` -> disabled.
    - `{"errorCode":100,"message":"Key not found","cause":"/message","index":<...>}` -> enabled.

My etcd version is 3.4.14.

Reply:
Updated.